### DISCUSSION TITLE
Release checklist

### DISCUSSION THREAD

**Discussion by tranzystorekk** [topgrade-rs/topgrade#156] [General]
Release-tagged commits tend to be missing a version bump in `Cargo.toml`/`Cargo.lock` by mistake, maybe a checklist somewhere in the repo could help?

**Comment by s34m** [comment]
In the future I would like to have a check which would require a version bump in the PR to master if something in src/ or the cargo files changes.

**Reply by tranzystorekk** [reply]
The semantic versioning workflow usually reserves a separate "chore: bump to version x.y.z" commit for version bumps, specifically for all the project files like `Cargo.toml` etc.

**Comment by GideonBear** [comment]
Ref #1141
